spec/requests/collections_spec.rb

```diff
@@ -235,19 +229,38 @@
       context 'when collection fails to save' do
         let(:collection_params) do
           {
-            collection: {
-              name: '',
-              depositor_sunets: ''
-            }
+            name: '',
+            depositor_sunets: ''
           }
         end
 
         it 'renders the page again' do
-          patch "/collections/#{collection.id}", params: collection_params
+          patch "/collections/#{collection.id}", params: { collection: collection_params, commit: 'Deposit' }
           expect(response).to have_http_status(:ok)
           expect(response.body).to include('All fields are required, unless otherwise noted.')
         end
       end
+
+      context 'with empty draft' do
+        let(:collection_params) do
+          {
+            name: '',
+            description: '',
+            contact_email: '',
+            access: 'world',
+            depositor_sunets: ''
+          }
+        end
+
+        xit 'creates a new draft collection' do
```

Comment:
this test should work based on it working in the browser, but doesn't...not sure why